_episodes/01-intro.md

```diff
@@ -132,3 +132,12 @@ so that her computer can work while she writes her paper.
 As a bonus,
 once she has put a processing pipeline together,
 she will be able to use it again whenever she collects more data.
+
+In order to achieve her task, Nelle needs to know how to:
+- navigate to a file/directory
+- create a file/directory
+- check the length of a file
+- chain processes together
```

Comment:
Just a suggestion for clarity. "commands" is used earlier in the episode, so that seems like a good substitute for the more generic "processes". "chain" could also be substituted for something like "connect", but I feel less strongly about that one.

```suggestion
- chain commands together
```

Reply:
Yep, good catch! I kinda like chain though because it has the connotation of a line of connected parts which follow the previous.